index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Microverse</title>
+    <title>Microverse.</title>
 </head>
 <body>
     <h1>Hellow Microverse</h1>
```

Comment:
## 3
### Your Summary
![pr-summary](https://user-images.githubusercontent.com/93039359/186142385-17521506-43b8-4aee-8bfb-16881a5508ad.png)

### Descriptive Summary Example :heavy_check_mark:
![descriptive-summary](https://user-images.githubusercontent.com/93039359/184921477-5686f05d-45cf-4d90-8522-6db2f012e302.png)

- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: